pytest_django/plugin.py

```diff
@@ -378,10 +378,10 @@ def _django_db_marker(request):
     This will dynamically request the ``db`` or ``transactional_db``
     fixtures as required by the django_db marker.
     """
-    marker = request.keywords.get('django_db', None)
```

Comment:
The two places like this bump the requirements to `pytest >= 3.6` - I think it would be easy enough to have code branch for `pytest < 3.6` to maintain existing compatibility if that feels like a blocker to this PR.

Reply:
No, I think requiring pytest 3.6 is fine.